Argo/Extensions/Dictionary.swift

```diff
@@ -20,6 +20,6 @@ extension Dictionary {
   }
 }
 
-func <^> <T, U, V>(@noescape f: T -> U, x: [V: T]) -> [V: U] {
+func <^> <T, U, V>( f: @noescape (T) -> U, x: [V: T]) -> [V: U] {
```

Comment:
ah, no we don't need them on operators ... but the automatic conversion tool left that extra space in there so I need to fix that